data/static/base_info.csv

```diff
@@ -61,7 +62,9 @@ United Nations World Food Programme (WFP),2013-06-11,Multilateral,wfp,70
 United States,2013-01-02,Government,unitedstates,27.25
 US Agency for International Development (USAID),2017-04-27,Government,usaid,33
 "World Bank, The",2011-03-14,Multilateral,worldbank,51.35
+World Health Organisation,2017-06-08,Multilateral,who,0
 World Vision International (WVI),2015-10-16,International NGO,wvi,6.25
 World Vision Netherlands,2016-06-20,National NGO,wvnld,3.25
 World Vision UK,2012-06-13,International NGO,wvuk,22
 World Vision Zimbabwe,2017-12-05,International NGO,wvz-gec,0
+ZOA,2016-02-10,International NGO,zoa,0
```

Comment:
Joined at the end of July 2017. Baseline value should be recalculated for 1 August 2017. It has been agreed that the first of the month will be taken for any new signatories.